RHDEVDOCS-6890: CQA 2.0: Work with builds (#100499)
openshift/openshift-docs: modules/ob-deleting-a-buildrun-resource.adoc

+
[source,terminal]
----
$ oc delete buildrun <buildrun_resource_name>
----
+
--
where:

`<buildrun_resource_name>`:: Specifies the name of the `buildrun` resource.
--

+
[source,terminal]
----
$ shp buildrun delete <buildrun_resource_name>
----
+
where:

`<buildrun_resource_name>`:: Specifies the name of the `buildrun` resource.

.Verification

* Verify that the `buildrun` resource is deleted by using the `oc` CLI or `shp` CLI:
+
[source,terminal]
----
$ oc get buildrun | grep <buildrun_resource_name>
----
+
--
where:

`<buildrun_resource_name>`:: Specifies the name of the `buildrun` resource.
--
If the `buildrun` resource is successfully deleted, the command will not return any output.
+
[source,terminal]
----
$ shp build list | grep <buildrun_resource_name>
----
+
--
where:

`<buildrun_resource_name>`:: Specifies the name of the `buildrun` resource.
--

> If the `buildrun` resource has been successfully deleted, the command will not return any output.

Reviewer: ```suggestion
--
+
If the `buildrun` resource is successfully deleted, the command does not return any output.
```

> If the `buildrun` resource has been successfully deleted, the command will not return any output.

Reviewer: Again, I would consider, to avoid repetition, to delete the first version of this and change the second to: ` . . . the commands do not return . . .`
